### ISSUE TITLE
Start path to dot directory ignored

### ISSUE THREAD

**Issue by noscript** [gokcehan/lf#167]
If the last path component is a dot directory it will be ignored, unless there is `/` at the end.

* `lf ~/.dot_dir` - will start in current working directory
* `lf ~/.dot_dir/` - will start in `~/.dot_dir`
* `lf ~/normal_dir` - will start in `~/normal_dir`

**Comment by gokcehan** [comment]
@noscript It used to be the case that when a directory is given as the first argument, `lf` would change to that directory, kind of like `ls`. We recently added file name selection as the first argument as well. The way we implemented this is to simply use `select` command to the first argument. It seems that `select` behavior is different for directory names with and without slash at the end. I think we may consider this a bug.

In your case, when `hidden` is not enabled, it simply start in the current working directory since it can't find the directory for selection. This is not the case when there is a slash at the end in which case it simply changes to that directory. Anyway, once we settled on the behavior of `select` command this issue should be resolved.
